readme.md

```diff
@@ -2,7 +2,7 @@
 
 This plugin supports a two-way integration between Mattermost and Jira. Jira Core and Jira Software products, for Server, Data Center, and Cloud platforms are supported. It has been tested with versions 7 and 8.
 
-From v3.0 of this plugin, support for multiple Jira instances is offered.
+From v3.0 of this plugin, support for multiple Jira instances is offered when using an Enterprise License.
```

Comment:
```suggestion
From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Server Enterprise Edition and Mattermost Cloud users.
```

Reply:
@jfrerich, is this correct?

Reply:
I think you could leave it a `for Mattermost Enterprise Editions`

Reply:
```suggestion
From v3.0 of this plugin, support for multiple Jira instances is offered for Mattermost Enterprise Editions.
```